rustfmt.toml

```diff
@@ -0,0 +1,66 @@
+max_width = 100
```

Comment:
Just running with defaults seems fine.

Reply:
The rustfmt.toml is generated using rustfmt with default settings.

Reply:
So then this file doesn't need to exist, right?

Reply:
while contributing to other rust opensource projects I got to know that - `without an included rustfmt.toml file runs the risk fo some things being formatted randomly at random times` reference - https://github.com/sagebind/sluice/pull/4#issuecomment-536084517, hence I included it.

Reply:
I'm not convinced that random format changes will happen.

Reply:
I am ok with not including rustfmt.toml in this PR, should I remove it?

Reply:
Yes

Reply:
done, I have updated the PR.